### ISSUE TITLE
Swift 4 Toolbar Siblings Array 

### ISSUE THREAD

**Issue by tsteele2013** [hackiftekhar/IQKeyboardManager#981]
App is crashing at line `2073` of `IQKeyboardManager.swift`. It looks like the most recent pull request #971 was supposed to fix this, but somehow it's not.

This issue only occurs when using the `navigationItem.searchController` now available in iOS 11. 

````
if siblings[0] == textField {
    if (siblings.count == 1) {
        textField.setEnablePrevious(false, isNextEnabled: false)
    } else {
        textField.setEnablePrevious(false, isNextEnabled: true)
    }
}
````

I'm getting the `2017-09-21 12:17:12.236089-0400 TruckTelematics[3091:1439388] fatal error: Index out of range` error.

Will try to fix this and submit a PR.

**Comment by hackiftekhar** [comment]
This is already fix with latest version 4.0.13 of library.

**Comment by tsteele2013** [comment]
@hackiftekhar Are you sure? When I switch to 4.0.13, I have 36 build errors in Swift 4. Perhaps those errors aren't related to my specific issue, but my project will not even build on 4.0.13.

**Comment by hackiftekhar** [comment]
The 4.0.13 isn’t swift4 version, that’s why you are getting compiler issue.

**Comment by tsteele2013** [comment]
Right, I understand that. My issue is with the `swift4` branch. And the PR I submitted is to fix the issue in Swift 4.

There might be a fix for this issue in `4.0.13`, but that doesn't help me since I'm trying to use the `swift4` branch. Does that make sense?

**Comment by hackiftekhar** [comment]
yes, there is a fix in `4.0.13` but might not have a fix in `swift4` branch, but next release of library will contain the `fix+iOS11+swift4`.

**Comment by tsteele2013** [comment]
Ahh, okay. Thank you for explaining.
